package.json

```diff
@@ -31,32 +31,33 @@
     "@vueuse/head": "^0.7.5",
     "ajv": "^8.10.0",
     "autoprefixer": "^10.4.4",
-    "biketag": "^3.2.7",
+    "biketag": "^3.0.0",
```

Comment:
It looks like these dependencies were downgraded. I'm not sure why but I will revert these changes.